plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+					'persistRender'    => true,
+				],
+			]
+		);
+		$product_inventory_advanced_wrapper = $product_inventory_advanced->add_block(
+			[
+				'blockName'  => 'woocommerce/product-section',
+				'attributes' => [
+					'blockGap' => 'unit-40',
+				],
+			]
+		);
+		$product_out_of_stock_conditional   = $product_inventory_advanced_wrapper->add_block(
+			[
+				'id'         => 'product-out-of-stock-conditional-wrapper',
+				'blockName'  => 'woocommerce/conditional',
+				'attributes' => [
+					'mustMatch' => [
+						'manage_stock' => [ true ],
+					],
+				],
+			]
+		);
+		$product_out_of_stock_conditional->add_block(
+			[
+				'id'         => 'product-out-of-stock',
+				'blockName'  => 'woocommerce/product-radio-field',
+				'attributes' => [
+					'title'    => __( 'When out of stock', 'woocommerce' ),
+					'property' => 'backorders',
+					'options'  => [
+						[
+							'label' => __( 'Allow purchases', 'woocommerce' ),
+							'value' => 'yes',
+						],
+						[
+							'label' => __(
+								'Allow purchases, but notify customers',
+								'woocommerce'
+							),
+							'value' => 'notify',
+						],
+						[
+							'label' => __( "Don't allow purchases", 'woocommerce' ),
+							'value' => 'no',
+						],
+					],
+				],
+			]
+		);
+		$product_out_of_stock_conditional->add_block(
+			[
+				'id'        => 'product-inventory-email',
+				'blockName' => 'woocommerce/product-inventory-email-field',
+			]
+		);
+
+		$product_inventory_advanced_wrapper->add_block(
+			[
+				'id'         => 'product-limit-purchase',
+				'blockName'  => 'woocommerce/product-checkbox-field',
+				'attributes' => [
+					'title'    => __(
+						'Restrictions',
+						'woocommerce'
+					),
+					'label'    => __(
+						'Limit purchases to 1 item per order',
+						'woocommerce'
+					),
+					'property' => 'sold_individually',
+					'tooltip'  => __(
+						'When checked, customers will be able to purchase only 1 item in a single order. This is particularly useful for items that have limited quantity, like art or handmade goods.',
+						'woocommerce'
+					),
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds the shipping group blocks to the template.
+	 */
+	private function add_shipping_group_blocks() {
+		$shipping_group = $this->add_group(
+			[
+				'id'         => 'shipping',
+				'order'      => 40,
+				'attributes' => [
+					'id'    => 'shipping',
+					'title' => __( 'Shipping', 'woocommerce' ),
+				],
+			]
+		);
+		$shipping_group->add_block(
+			[
+				'id'         => 'product_variation_notice_shipping_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
+					'content'    => __( 'This product has options, such as size or color. You can now manage each variation\'s price and other details individually.', 'woocommerce' ),
+					'buttonText' => __( 'Go to Variations', 'woocommerce' ),
+					'type'       => 'info',
+				],
```

Comment:
The `blockName` seems pretty specific. Yet we are passing all these attributes. It seems like either the `woocommerce/product-has-variations-notice` block is more generic than it sounds like it should be, or the attributes shouldn't need to be passed.

Reply:
Yeah I agree, you can partially blame me, I was the reviewer on that one and end up letting it slide. That PR went through several iterations, where one option was a generic notice and the other option a more specific one. Turns out it end up being a mix of both, as the block it-self contains specific hide/show logic if options exist.
I let it slide, as it doesn't hurt having access to the text through the attributes, but blocks like these may warrant a wider discussion on how `notices` of this kind should work.

Reply:
But this is un-related to this PR, so we can create a follow up issue if you think it should be one or the other.

Reply:
Issue created: https://github.com/woocommerce/woocommerce/issues/39870